examples/tutorials/31_Introduction_To_Material_Science.ipynb

```diff
@@ -0,0 +1,428 @@
+   "source": [
+    "!pip install --pre deepchem"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cbec9313",
+   "metadata": {},
+   "source": [
+    "DeepChem for material science will also require the additiona libraries [`pymatgen`](https://pymatgen.org/) and [`matminer`](https://hackingmaterials.lbl.gov/matminer/). These two libraries assist machine learning in material science. For graph neural network models which we will be used in the backend, DeepChem requires [`dgl`](https://www.dgl.ai/) library. All these can be installed using `pip`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "f162536e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install pymatgen~=2020.12\n",
+    "!pip install matminer==0.6.5\n",
+    "!pip install dgl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "3335e96f",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'2.6.0.dev'"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import deepchem as dc\n",
+    "dc.__version__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "6f76f7a5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pymatgen as mg\n",
+    "\n",
+    "import os\n",
+    "os.environ['DEEPCHEM_DATA_DIR'] = os.getcwd()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d43c4bc3",
+   "metadata": {},
+   "source": [
+    "## Featurizers <a class=\"anchor\" id=\"featurizers\"></a>\n",
+    "\n",
+    "### Material Structure Featurizers <a class=\"anchor\" id=\"crystal-featurizers\"></a>\n",
+    "\n",
+    "Crystal are gemoetric structures which has to be featurized for using in machine learning algorithms.  The following featurizers provided by DeepChem helps in featurizing crystals:\n",
+    "\n",
+    "- The [SineCoulombMatrix](https://deepchem.readthedocs.io/en/latest/api_reference/featurizers.html#sinecoulombmatrix) featurizer a crystal by calculating sine coulomb matrix for the crystals. It can be called using `dc.featurizers.SineCoulombMatrix` function.\n",
+    "- The [CGCNNFeaturizer](https://deepchem.readthedocs.io/en/latest/api_reference/featurizers.html#cgcnnfeaturizer) calculates structure graph features of crystals. It can be called using `dc.featurizers.CGCNNFeaturizer` function. For more details on CGCNNFeaturizer, refer [1].\n",
+    "- The [LCNNFeaturizer](https://deepchem.readthedocs.io/en/latest/api_reference/featurizers.html#lcnnfeaturizer) calculates the 2-D Surface graph features in 6 different permutations. It can be used using the utility `dc.feat.LCNNFeaturizer`.\n",
+    "\n",
+    "[1] T. Xie and J. C. Grossman, “Crystal graph convolutional neural networks for an accurate and interpretable prediction of material properties”, Phys. Rev. Lett. 120, 2018, https://arxiv.org/abs/1710.10324"
```

Comment:
Can you add a citation for lattice convolutions to this paper: https://pubs.acs.org/doi/10.1021/acs.jpcc.9b03370